go/host/storage/hostdb/sql_statements.go

```diff
@@ -0,0 +1,39 @@
+package hostdb
+
+// SQLStatements struct holds SQL statements for a specific database type
+type SQLStatements struct {
+	InsertBatch        string
+	InsertTransactions string
+	InsertTxCount      string
+	InsertRollup       string
+	InsertBlock        string
+	SelectRollups      string
+	SelectBlocks       string
+	Placeholder        string
+}
+
+func SQLiteSQLStatements() *SQLStatements {
```

Comment:
Quite annoying we have to maintain two syntaxes, but it is what it is.

Reply:
we were lucky with the enclave db syntaxs not being different tbh